File: criver-client/handlers-template/basic-repo-event.py
#! /usr/bin/python3

### ------------------------------------
####  what does this event-handler it does?: ####

# It print the type of all repository events we receive

import os

# This handler looks if a repository in GitHub has a new tag event(CreateEvent).
## 1) open the tmp data file
# open the repo-events json. The env variable contain the file full-path



criver_file = os.environ['CRIVER_EVENTS_REPO']
with open(criver_file) as json_file:  
    data = json.load(json_file)
    print(data['type'])

File: criver-client/handlers-template/basic-repo-event.rb
#! /usr/bin/ruby

require 'json'

# through env var read info about repo
file = File.open(ENV['CRIVER_EVENTS_REPO'])
criver_data = JSON.load file

# do some operations with the data..
puts criver_data[0]
puts criver_data[0]["type"]
